packages/frontend/src/components/SimplePieChart/PieChartContextMenu.tsx

```diff
@@ -156,10 +160,11 @@ const PieChartContextMenu: FC<PieChartContextMenuProps> = ({
                     Copy
                 </Menu.Item>
 
-                {canViewUnderlyingData ? (
+                {canViewUnderlyingData && !hasCustomDimension(metricQuery) ? (
                     <Menu.Item
                         icon={<MantineIcon icon={IconStack} />}
                         onClick={handleOpenUnderlyingDataModal}
+                        title="Underlying data is not available with custom dimensions."
```

Comment:
we don't need the title since is not visible

Reply:
Yeah, sorry, a leftover from testing